README.md

```diff
@@ -16,7 +16,7 @@ at the beginning of the session.
 ## Path of vocabulary file per OS
 
 * Linux: `~/.config/Vocabulary Test/vocabulary.txt`
-* Mac: :question:
+* Mac: `/Users/{USER_NAME}/Library/Application\ Support/Vocabulary\ Test/`
```

Comment:
Please use the `~` character to address the user home directory:
```suggestion
* Mac: `~/Library/Application\ Support/Vocabulary\ Test/`
```

Reply:
Gute Idee :-)